packages/analysis/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "noImplicitAny": true,
```

Comment:
This is stricter than Terra UI, so some code may have to be updated when it's moved over.
https://www.typescriptlang.org/tsconfig#noImplicitAny